candle-book/src/guide/hello_world.md

```diff
@@ -1 +1,192 @@
-# PyTorch cheatsheet
+# Hello world !
+
+We will now create the hello world of the ML world, building a model capable of solving MNIST dataset.
+
+Open `src/main.rs` and fill in with these contents:
```

Comment:
these contents -> this content?
Also maybe we should have a repo with the actual files so that it's easier for people that just want to copy the files around?

Reply:
Fixed typo.

I'm hesitant for the final repo, I'm sure we're not going to keep it enough up to date and currently there's very little content.
But maybe later when the doc grows( and we could inner include too).